generate.go

```diff
@@ -46,21 +47,10 @@ type DependencyManager interface {
 
 func Generate(dependencyManager DependencyManager, logger scribe.Emitter) packit.GenerateFunc {
 	return func(context packit.GenerateContext) (packit.GenerateResult, error) {
-		logger.Title("%s %s", context.Info.Name, context.Info.Version)
-		logger.Process("Resolving Node Engine version")
 
-		// likely move this out to main
 		entryResolver := draft.NewPlanner()
+		entry, _ := libnodejs.ResolveNodeVersion(entryResolver.Resolve, context.Plan);
```

Comment:
Did you still want to keep `allEntries` around for logging purposes or was this intentional?

Reply:
Good, catch I rebased on Costas PR which landed recently and fixed that.